src/graph.jl

```diff
@@ -24,17 +24,7 @@ end
 SparsityPatternCSC(A::SparseMatrixCSC) = SparsityPatternCSC(A.m, A.n, A.colptr, A.rowval)
 
 Base.size(S::SparsityPatternCSC) = (S.m, S.n)
-
-function Base.size(S::SparsityPatternCSC, d::Integer)
-    if d == 1
-        return S.m
-    elseif d == 2
-        return S.n
-    else
-        return 1
-    end
-end
-
+Base.size(S::SparsityPatternCSC, d) = d::Integer <= 2 ? size(S)[d] : 1
```

Comment:
1 <= d::Integer <= 2 🙂

Reply:
À part si tu laisses Julia faire une erreur volontairement

Reply:
Funnily enough, this is exactly what they do in `Base`, I literally copy-pasted this line
https://github.com/JuliaLang/julia/blob/fbb3e1175d52abec0ff4ca83d8c9e126d9f8a06b/base/abstractarray.jl#L42